### DISCUSSION TITLE
Request entity too large on post

### DISCUSSION THREAD

**Discussion by Adamekjr** [remix-run/remix#7175] [Q&A]
Hi, I am having an issue while trying to post form larger than 100kb.
There is some limit here in raw-body/index.js
![image](https://github.com/remix-run/remix/assets/91897829/50e4250a-1180-41bc-aae1-bbce55e78e3f)
Is there any possibility to extend this limit?

**Comment by kiliman** [comment]
I do not see that code anywhere in the Remix repo. Do you have a simple repo?

**Comment by Adamekjr** [comment]
I am using Remix with Hydrogen Shopify. It's possible that I have opened it in the wrong location.
That file is in node modules.

**Reply by davidmoreiragaia** [reply]
Were you able to solve this problem ?

**Reply by Adamekjr** [reply]
Basically no, it won't work in development env. When it's deployed to production, then it's fine and there is no limit.
